pkg/pillar/scripts/device-steps.sh

```diff
@@ -165,6 +165,11 @@ echo "$(date -Ins -u) device-steps: Starting upgradeconverter"
 status=$($BINDIR/upgradeconverter)
 echo "$(date -Ins -u) device-steps: upgradeconverter Completed. Status: $status"
 
+echo "$(date -Ins -u) device-steps: Creating additional security certificates"
+if ! $BINDIR/tpmmgr createCerts; then
```

Comment:
How did these get created up until now?

And what happens on an existing device where we have 4 of the 5 keys and only need to create the new one?
Are the functions robust and retrieve the existing 4 keys in that case?

Reply:
These were created from RunAsService() method, I am moving it out to a separate call in device-steps.sh, to align with the objective of freezing /config partition after initial setup.  

If we have 4 out 5 keys, the method will create just the missing key (or cert).

Reply:
ok